c/zss.c

```diff
@@ -317,10 +319,38 @@ static void setPrivilegedServerName(HttpServer *server, JsonObject *mvdSettings,
 }
 #endif /* __ZOWE_OS_ZOS */
 
+static int nativeWithSessionTokenAuth(HttpService *service, HttpRequest *request, HttpResponse *response) {
+  int rc = 0;
+  char profileName[ZOWE_PROFILE_NAME_LEN+1] = {0};
+  char method[16];
+  snprintf(method, sizeof(method), "%s", request->method);
+  destructivelyNativize(method);
+  rc = getProfileNameFromRequest(profileName, request->parsedFile, method, -1, response);
+  if (rc != 0) {
+    return FALSE;
+  }
+  rc = serveAuthCheckByParams(service, request->username, "ZOWE", profileName, 2, ENV_SETTINGS);
```

Comment:
What does `2` (and `-1` in the previous call) mean? Why is it not an enum or `#define`?